LayoutTests/fast/images/decode-static-svg-image-resolve-expected.html

```diff
@@ -0,0 +1,22 @@
+<style>
+/* This tests passes if two green rectangles are drawn. */
+    .rectangles {
+        display: inline-block;
+    }
+    .rectangle {
+        width: 100px;
+        height: 100px;
+        display: inline-block;
+    }
+    .green {
+        background-color: green;
+    }
+</style>
+<body>
+    <div class="container">
```

Comment:
What is `container`? Do we really need this `<div>`?

Reply:
Removed it.